config/samples/resources/cloudidsendpoint/cloudids_v1beta1_servicenetworkingconnection.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Google LLC
```

Comment:
The name should be `servicenetworking_v1beta1_servicenetworkingconnection.yaml`.